v0/destinations/monday/util.js

```diff
@@ -0,0 +1,242 @@
+const { isNumber } = require("lodash");
+const { httpPOST } = require("../../../adapters/network");
+const {
+  processAxiosResponse
+} = require("../../../adapters/utils/networkUtils");
+const { CustomError, getDestinationExternalID } = require("../../util");
+
+// regex to validate latitude and longitude
+const regexExp = /^((\-?|\+?)?\d+(\.\d+)?),\s*((\-?|\+?)?\d+(\.\d+)?)$/gi;
```

Comment:
Where are we using this? I am not seeing any other reference.

Reply:
Removed it, it was being used previously.